src/app/app.module.ts

```diff
@@ -1,10 +1,22 @@
 import { BrowserModule } from '@angular/platform-browser';
 import { NgModule } from '@angular/core';
+import { RouterModule, Routes } from '@angular/router';
 import { FormsModule } from '@angular/forms';
 import { HttpModule } from '@angular/http';
 import { AngularFireModule } from 'angularfire2';
 
 import { AppComponent } from './app.component';
+import { HeaderComponent } from './header/header.component';
+import { ProfileComponent } from './profile/profile.component';
+import { CommunityComponent } from './community/community.component';
+import { DashboardComponent } from './dashboard/dashboard.component';
```

Comment:
You'll need to add your components and.. you'll know if you'll need more later.. it will fail lol